GoogleForCreators/web-stories-wp
# Ensure selection is displayed on top of Page navigation arrows

## Summary
Moves the page nav to the same level with page menu and selection so that it could be layered as expected.


## Relevant Technical Choices



## To-do



## User-facing changes
- Selection now appears on top of the page navigation arrows.


## Testing Instructions
1. Add a new Page
2. Add an element
3. Resize and move the element so that the rotation handle would be on top of the page navigation arrow
4. Verify that the rotation handle is on top of the arrow and can still be used


---



Fixes #2814
